proto/pki/pki_revocation_distribution_points_by_issuer_subject_key_id.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+package zigbeealliance.distributedcomplianceledger.pki;
+
+option go_package = "github.com/zigbee-alliance/distributed-compliance-ledger/x/pki/types";
+
+message PkiRevocationDistributionPointsByIssuerSubjectKeyID {
+  string issuerSubjectKeyID = 1; 
+  
```

Comment:
`repeated PkiRevocationDistributionPoint points = 2;` is missed (the actual content)